docs/source/models/causes/neonatal/index.rst

```diff
@@ -158,32 +175,57 @@ unit time.
-      - mortality risk
-      - The probability that a simulant who was born alive dies during the neonatal period
+    * - enn_mr
+      - mortality risk during the early neonatal period
+      - The probability that a simulant who was born alive dies during the first 7 days
+    * - lnn_mr
+      - mortality risk during the late neonatal period
+      - The probability that a simulant who was born alive dies between day 8 to 28 of life
 
 
 Data Tables
 +++++++++++
 
-The neonatal death model requires only the probability of death (aka "mortality risk") for use
-in the decision graph. This will be computed as
+The neonatal death model requires only the probability of death (aka "mortality risk") for the early and late neonatal time periods.  But computing this for an individual simulant is a bit complicated.  It will follow the pattern from the general mortality component in :code:`vivarium_public_health`, and work in rate space to make the math simpler.  The final step will be converting from rates to risks.
+
+.. math::
+    \begin{align*}
+    \text{mr}_i &= 1-\exp(-\text{ACMR}_i \Delta t),
+    \end{align*}
+
+where :math:`\text{mr}_i` is the probability of mortality for simulant :math:`i` during the early or late neonatal period, :math:`\text{ACMR}_i` is the all-cause mortality rate for the early or late neonatal period, and :math:`\Delta t` is length of that period in years.
+
+The calculation of :math:`\text{ACMR}_i` is a bit complicated, however. We begin with a population ACMR and use the LBWSG PAF to derive a risk-deleted ACMR to which we can then apply the relative risk of LBWSG matching each risk exposure category.  Mathematically this is achieved by the following formula:
+
+.. math::
+    \begin{align*}
+    \text{ACMR}_j &= \text{ACMR} \times (1 - \text{PAF}_{\text{LBWSG}}) \times \text{RR}_{\text{LBWSG},j},
+    \end{align*}
+
+where :math:`\text{ACMR}_j` is the all-cause mortality rate for the population with LBWSG exposure category :math:`j`, :math:`\text{ACMR}` is the all-cause mortality rate for the total population, :math:`\text{PAF}_{\text{LBWSG}}` is the population attributable fraction for LBWSG, and :math:`\text{RR}_{\text{LBWSG}}` is the relative mortality rate for LBWSG.
+
+To obtain the ACMR for a specific simulant, we subtract off the *population* CSMRs for each modeled subcause for the LBWSG exposure level of the simulant, and then add back in the (potentially pipeline-modified) *individual* CSMRs for the specific simulant, which might differ from baseline due to intervention coverage.
 
 .. math::
     \begin{align*}
-    \text{mr} &= \frac{\text{neontal deaths}}{\text{live births}}\\
-        &= \frac{\text{(neonatal mortality rate)} \cdot  \text{(neonatal person-time)}}
-            {\sum_{\text{age groups}} \text{ASFR} \cdot (\text{age-specific person-time})
-            }.
+    \text{ACMR}_i &= \text{ACMR}_j - \sum_k \text{CSMR}_{k,j}^{\text{population}}
+    + \sum_k \text{CSMR}_{k,i},^{\text{individual}}
```

Comment:
Ah, so this is what you meant when you said we could modify the EMRs (or CSMRs) instead of having the intervention revive the simulant after we decide they're going to die, right?